nas.yml

```diff
@@ -226,6 +226,11 @@
         - mosquitto
       when: (mosquitto_enabled | default(False))
 
+    - role: mumble
+      tags:
+        - mumble
+      when: (mumble_enabled | default(False))
```

Comment:
This line will need to be removed or the `mumble_enabled == false' block won't ever get run in the main.yml file.

Reply:
Removed.